CMakeLists.txt

```diff
@@ -83,9 +83,13 @@ add_dependencies(taihen-libs taihen.elf)
 install(DIRECTORY ${CMAKE_BINARY_DIR}/taihen-libs/
 		DESTINATION lib
 		FILES_MATCHING PATTERN "*.a")
-		
+
 install(FILES ${CMAKE_BINARY_DIR}/taihen.json
 		DESTINATION share/imports)
 
 install(FILES taihen.h
 		DESTINATION include)
+
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+	set(CMAKE_INSTALL_PREFIX "$ENV{VITASDK}/arm-vita-eabi")
```

Comment:
Should we also look for $ENV{VITASDK} to be set? Otherwise won't it default to blank?